### DISCUSSION TITLE
Implement CMD42 in SD_MMC Library

### DISCUSSION THREAD

**Discussion by msauvayre** [espressif/arduino-esp32#10054] [Q&A] [answered]
Hi everyone,

I'm currently develop a forensic project for engineer student. I've started a project based on an ESP32 CAM which is embed a microSD card slot.

I have a good knowledge about the MMC protocol, but so far, I've not seen any mention about SD Card locking/CMD42 in the library.

So, I'm wondering if it's possible to add the CMD42's management inside the library by myself of if the developer plan to add it.

Thanks by advance for any answer.
Regards.

**Comment by me-no-dev** [comment, marked as answer]
Our knowledge about the SD protocol is probably less than yours. If you think that this would improve the library, please feel free to submit a PR

**Reply by msauvayre** [reply]
Thanks for your answer. I will try to add it as soon as I can.
